config.toml.example

```diff
@@ -523,7 +523,9 @@ changelog-seen = 1
 #test-compare-mode = false
 
 # Use LLVM libunwind as the implementation for Rust's unwinder.
-#llvm-libunwind = false
+# Accepted values are 'in-tree' (formerly true) or 'system'.
+# Any other value is ignored (formerly false).
```

Comment:
We should definitely not be silently accepting arbitrary values; I would expect either an explicit "false" (either literally bool or `"false"` or `"no"` to be required. (And the default being parsed as false, of course).

Reply:
Went with `"no"` so that it's obvious that a string is expected and not a bool. Not sure if there's a cleaner way to reject an invalid config value right now, I don't think there is such a validation for other fields